ocs_ci/ocs/resources/cloud_uls.py

```diff
@@ -105,15 +105,16 @@ def _create_uls(uls_dict):
 
     def uls_cleanup():
         for cloud, uls_set in all_created_uls.items():
-            client = ulsMap.get(cloud)
-            if client is not None:
-                all_existing_uls = client.get_all_uls_names()
-                for uls in uls_set:
-                    if uls in all_existing_uls:
-                        log.info(f"Cleaning up uls {uls}")
-                        client.delete_uls(uls)
-                    else:
-                        log.warning(f"Underlying Storage {uls} not found.")
+            if len(uls_set) > 0:
```

Comment:
It would be more readable if you just do "if len(uls_set) == 0: continue" instead (avoiding too nested branching).

Reply:
Awesome and elegant suggestion, thank you!